proxy/proxy.go

```diff
@@ -213,6 +221,12 @@ func mapRequest(r *http.Request, rt *routing.Route, host string) (*http.Request,
 	u.Scheme = rt.Scheme
 	u.Host = rt.Host
 
+	backendURL, err := url.Parse(rt.Backend)
```

Comment:
a topic of ongoing discussion...
but since I see that now the filters are executed on the request flow, we could use the basicAuth filter instead of putting the user and the password in the backend address. Note that the backend address is limited by design to scheme and host (https://godoc.org/github.com/zalando/skipper/eskip#hdr-Backend)